### ISSUE TITLE
Dusk not screenshot when the test fail

### ISSUE THREAD

**Issue by pr0ph0z** [laravel/dusk#630]
Hi there, I'm currently new to Dusk.
I wrote the simple test to make it fail. Here's the code
```
<?php

namespace Tests\Browser;

use Tests\DuskTestCase;
use Laravel\Dusk\Browser;
use Illuminate\Foundation\Testing\DatabaseMigrations;

class PerusahaanTest extends DuskTestCase
{
    /**
     * A Dusk test example.
     *
     * @return void
     */
    public function testExample()
    {
        $this->browse(function (Browser $browser) {
            $browser->visit('http://localhost:8000/')
                    ->assertSee('Laravels');
        });
    }
}
```
When I run `php artisan dusk`, there's an error
```
Warning: TTY mode is not supported on Windows platform.
PHPUnit 7.5.7 by Sebastian Bergmann and contributors.
DevTools listening on ws://127.0.0.1:52448/devtools/browser/85c71994-ab97-4753-95bb-14dc7ebd405b                        E                                                                   1 / 1 (100%)
Time: 3.73 seconds, Memory: 14.00 MB
There was 1 error:
1) Tests\Browser\PerusahaanTest::testExample
ErrorException: file_put_contents(C:\Users\rds\dev\lpse\tests/Browser/screenshots/failure-Tests_Browser_PerusahaanTest_testExample-0.png):
failed to open stream: No such file or directory
C:\Users\rds\dev\lpse\vendor\facebook\webdriver\lib\Remote\RemoteWebDriver.php:348
C:\Users\rds\dev\lpse\vendor\laravel\dusk\src\Browser.php:274
C:\Users\rds\dev\lpse\vendor\laravel\dusk\src\Concerns\ProvidesBrowser.php:139
C:\Users\rds\dev\lpse\vendor\laravel\framework\src\Illuminate\Support\Collection.php:419
C:\Users\rds\dev\lpse\vendor\laravel\dusk\src\Concerns\ProvidesBrowser.php:140
C:\Users\rds\dev\lpse\vendor\laravel\dusk\src\Concerns\ProvidesBrowser.php:69
C:\Users\rds\dev\lpse\tests\Browser\PerusahaanTest.php:21
ERRORS!
Tests: 1, Assertions: 1, Errors: 1.
```
There should be an image saved but it shows error. Am i missed something?

**Comment by staudenmeir** [comment]
Do you have a `screenshots` directory in `C:\Users\rds\dev\lpse\tests\Browser`?

Did you run `php artisan dusk:install`?

**Comment by pr0ph0z** [comment]
@staudenmeir oops, i didn't run the `php artisan dusk:install`, my bad. thank you :).

**Comment by edstevo** [comment]
I get this issue using Laravel Sail. 

ErrorException: file_put_contents(/var/www/html/tests/Browser/screenshots/failure-Tests_Browser_ExampleTest_testBasicExample-0.png): Failed to open stream: No such file or directory

I can confirm:
1) Followed installation instructions on Laravel website, and noted suggestions when using Sail,
2) Installed on a fresh install of laravel as a test,
3) Directory "/var/www/html/tests/Browser/screenshots" exists,
4) I ran php artisan dusk:install.

As I say, I've tested this inside my project and this morning I tried a fresh install. Same problem for both.

What am I missing?
